.github/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+https://jira.percona.com/browse/PMM-0
```

Comment:
Autolinks are unavailable for Percona-Lab namespace, so I used the full link here.